package.json

```diff
@@ -111,14 +111,17 @@
     ]
   },
   "scripts": {
-    "vscode:prepublish": "tsc -p ./",
-    "compile": "tsc -watch -p ./",
-    "preinstall": "node ./scripts/preinstall.js",
+    "vscode:prepublish": "npm run compile",
+    "compile": "tsc -p ./",
+    "watch": "tsc -watch -p ./",
+    "preinstall": "npm i --save mvn-artifact-download && node ./scripts/preinstall.js",
     "postinstall": "node ./node_modules/vscode/bin/install",
     "test": "node ./node_modules/vscode/bin/test",
-    "tslint": "gulp tslint"
+    "tslint": "gulp tslint",
+    "ui-test": "npm run compile && extest setup-and-run --code_version 1.38.1 out/ui-test/*.js -o src/ui-test/settings.json"
```

Comment:
Should we let it using the latest version?
or if we specify a version maybe we need to have itr as a parameter so that we can test severla versions directly on Ci? (or it might be overengineered and better to handle it in another iteration)

I'm thinking loud :-)

Reply:
yeah, very good question. I think zou are right I can remove '--code_version' parameter for now and let it uses latest version (which is currently same 1.38.1). For next iteration I like your idea to have it parametrized for testing lets say latest two or three vscode versions back. We will also see how much will allow us new version of vscode-extension-tester.

Reply:
ok, I vote for remove the --code_version for now. As VS Code is upgrading automatically by default it will allow us to catch issues relatively fast.
I let you create an issue for another iteration with several version to be tested.
I see it well as different travis stage build parameterized with:
- the minimally declared version
- the latest stable version
- the "insiders" build

Reply:
removed '--code_version' -> using latest version of vscode